backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/optimizations/EscapeAnalysis.kt

```diff
@@ -1661,13 +1664,27 @@ internal object EscapeAnalysis {
                          callGraph: CallGraph, lifetimes: MutableMap<IrElement, Lifetime>) {
         assert(lifetimes.isEmpty())
 
-        val intraproceduralAnalysisResult =
-                IntraproceduralAnalysis(context, moduleDFG, externalModulesDFG, callGraph).analyze()
-        InterproceduralAnalysis(context, callGraph, intraproceduralAnalysisResult, externalModulesDFG, lifetimes,
-                // TODO: This is a bit conservative, but for more aggressive option some support from runtime is
-                // needed (namely, determining that a pointer is from the stack; this is easy for x86 or x64,
-                //         but what about all other platforms?).
-                propagateExiledToHeapObjects = true
-        ).analyze()
+        try {
+            val intraproceduralAnalysisResult =
+                    IntraproceduralAnalysis(context, moduleDFG, externalModulesDFG, callGraph).analyze()
+            InterproceduralAnalysis(context, callGraph, intraproceduralAnalysisResult, externalModulesDFG, lifetimes,
+                    // TODO: This is a bit conservative, but for more aggressive option some support from runtime is
+                    // needed (namely, determining that a pointer is from the stack; this is easy for x86 or x64,
+                    //         but what about all other platforms?).
+                    propagateExiledToHeapObjects = true
+            ).analyze()
+        } catch (t: Throwable) {
+            val extraUserInfo =
+                        """
+                        Please try to disable escape analysis and rerun the build. To do it add "-Xdisable-phases=EscapeAnalysis" to the compilation command line or for Gradle, as an example:
+
+                            macosX64("macos") {
```

Comment:
Maybe this instead?
```
kotlin {
    targets.withType<org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget> {
        binaries.all {
            freeCompilerArgs += "-Xdisable-phases=EscapeAnalysis"
        }
    }
}
```
It will allow user to trivially copy-paste this fragment to `build.gradle.kts`.